### DISCUSSION TITLE
spring/kotest/

### DISCUSSION THREAD

**Discussion by giscus** [ambosing/ambosing.github.io#3] [Blog-Comments]
# spring/kotest/

스프링부트 테스트 정복해보자구

https://ambosing.github.io/spring/kotest/

**Comment by yunjae228** [comment]
감사합니다 ! 덕분에 테스트 코드를 성공적으로 작성할 수 있었어요 !

**Comment by mingeonho1** [comment]
Junit 말고도 kotest를 알게되어 좋았습니다 !
